packages/create-pantheon-decoupled-kit/src/index.ts

```diff
@@ -85,11 +85,29 @@ export const main = async (
 	// If no generators are found in positional params
 	// ask which generators should be run
 	if (!foundGenerators || !foundGenerators.length) {
-		const whichGenerators: QuestionCollection<{ generators: string[] }> = {
+		// check if a cmsType was provided
```

Comment:
There should probably be some feedback in case the user chooses an invalid cms type like `wordpress` or `WordPress`, or even typos like `drpual`.  Check if the type is valid, and if not log a message that says something like `"invalid cmsType: ${cmsType}. Showing all generators"`.